flipperdevices/flipperzero-firmware
# [FL-2797] Signal Generator app

# What's new

- Signal generator app prototype, currently supports: 
  - Generating PWM using TIM1 or LPTIM2 outputs
  - Configurable clock output to MCO pin
- furi_hal_pwm
- furi_hal_clock: MCO configuration

# Verification 

- Build and upload plugins
- Check Tools/Signal Generator

# Checklist (For Reviewer)
